cmd/aws-iam-authenticator/verify.go

```diff
@@ -79,6 +84,24 @@ var verifyCmd = &cobra.Command{
 	},
 }
 
+// Uses EC2 metadata to get the region. Returns "" if no region found.
+func getInstanceRegion(ctx context.Context) string {
+	cfg, err := config.LoadDefaultConfig(ctx)
+	if err != nil {
+		fmt.Printf("[Warn] Unable to create metadata client, err: %v", err)
```

Comment:
nit: the error message is not correct right as we just generating config here  and not creating client?